src/components/commentary/Commentary.js

```diff
@@ -445,13 +453,13 @@ const Commentary = (props) => {
         <h5 className={classes.message}>
           {message === "loading"
             ? "Loading"
-            : `No commentary available for ${book} ${chapter}`}
+            : t("studyNoCommentaryAvailable", { ref })}
         </h5>
       )}
       {commentaryIntro.bookIntro && (
         <div onClick={toggleIntro} className={classes.introHeading}>
           <Typography className={classes.introTitle}>
-            Introduction to {book}
+            {t("studyCommIntroTo")} {book}
```

Comment:
Add interpolation here 
```suggestion
            {t("studyCommIntroTo", { book })}
```